presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/BigIntegerAndLongStateSerializer.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.state;
+
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.block.BlockBuilder;
+import com.facebook.presto.spi.type.Type;
+import io.airlift.slice.DynamicSliceOutput;
+import io.airlift.slice.SliceInput;
+
+import static com.facebook.presto.spi.type.LongDecimalType.unscaledValueToBigInteger;
+import static com.facebook.presto.spi.type.LongDecimalType.unscaledValueToSlice;
+import static com.facebook.presto.spi.type.VarbinaryType.VARBINARY;
+import static io.airlift.slice.SizeOf.SIZE_OF_LONG;
+
+public class BigIntegerAndLongStateSerializer
+        implements AccumulatorStateSerializer<BigIntegerAndLongState>
+{
+    @Override
+    public Type getSerializedType()
+    {
+        return VARBINARY;
```

Comment:
I'm not sure what difference it makes whether I work with `VARBINARY` here, `VARCHAR` or other type using `Slice`.